### ISSUE TITLE
Error during uninstalling OneDrive

### ISSUE THREAD

**Issue by grandtiger** [10se1ucgo/DisableWinTracking#405]
**INFORMATION:**

* Windows 10 Pro 1803
* Architecture: 64bit
* Options used: Uninstall OneDrive, Service Method Delete, Mode Privacy

**LOG:**

```python
12:19:03 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
12:19:03 INFO: ('Windows', 'T420SWIN', '10', '10.0.17134', 'AMD64', 'Intel64 Family 6 Model 42 Stepping 7, GenuineIntel')
12:19:03 INFO: DisableWinTracking version 3.2.1
12:19:19 INFO: Registry: Successfully modified ListPin key.
12:19:19 INFO: Registry: Successfully modified ListPin64Bit key.
12:19:19 INFO: Registry: Successfully modified FileSync key.
12:19:21 INFO: OneDrive: unable to uninstall. Exited with code: 0 - ('', '')
12:19:21 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
12:19:21 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions
```

**Comment by Ruined1** [comment]
Microsoft changed the location of the onedrive setup file. Surprise, surprise. I'll get that resolved in the next update.

**Comment by Ruined1** [comment]
Fixed in 3.2.2
```
19:44:44 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)] on win32
19:44:44 INFO: ('Windows', 'User', '10', '10.0.17763', 'AMD64', 'AMD64 Family 23 Model 8 Stepping 2, AuthenticAMD')
19:44:44 INFO: DisableWinTracking version 3.2.2
19:44:47 INFO: Registry: Successfully modified ListPin key.
19:44:47 INFO: Registry: Successfully modified ListPin64Bit key.
19:44:47 INFO: Registry: Successfully modified FileSync key.
19:45:11 INFO: OneDrive: successfully uninstalled
19:45:11 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
19:45:11 INFO: If you feel something didn't work properly, please press the 'Report an issue' button and follow the directions
```

**Comment by Ruined1** [comment]
Actually, I believe it's because you're using 32-bit in hind-sight. I'll need to review where the file is located on a 32 bit system and figure that out.

**Comment by Ruined1** [comment]
Fixed in 3.2.2
